requirements.txt

```diff
@@ -0,0 +1 @@
+garth==0.4.28
```

Comment:
It seems that 0.4.28 doesn't exist everywhere (I could install it locally, but [github action](https://github.com/pe-st/garmin-connect-export/actions/runs/6337921408/job/17322992456?pr=96) fails with `Could not find a version that satisfies the requirement garth==0.4.28 (from versions: 0.4.29, 0.4.30, 0.4.31, 0.4.32, 0.4.33, 0.4.34, 0.4.35, 0.4.36)`)

Is there a way in `requirements.txt` to specify "latest" or 0.4.x, as garth has seen a couple of releases since opening the PR ;-)